Car-Specs/Car-Specs/appsettings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "ConnectionUrl": "mongodb+srv://vasanth19230210:qCybr32y1xPEfcu8@car-specs.dzvzzcy.mongodb.net/",
```

Comment:
you can use user-secrets, avoid committing to git

Reply:
Changed to user-secrets